pkg/p2p/kadcast/network.go

```diff
@@ -0,0 +1,55 @@
+package kadcast
+
+import (
+	"log"
+	"net"
+)
+
+// Listens infinitely for UDP packet arrivals and 
+// executes it's processing inside a gorutine.
+func startUDPListener(netw string, router *Router) {
+	lAddr := getLocalIPAddress()
+	// Set listening port.
+	lAddr.Port = int(router.myPeerInfo.port)
+	log.Printf("%v", lAddr)
```

Comment:
When logging, it's more idiomatic to specify the fields seperately from the message, and assigning it a specific level. An example:
```
log.WithFields(log.Fields{
        "address": lAddr,
        "process": "kadcast",
}).Debugln("starting UDP listener")
```

Reply:
Currently the logs are only for debugging purposes, once we have a simple-working implementation, we can add the Logs with this format.

Thanks!

Reply:
Which log package are you using? Since the one I use does not have this methods.